lib/resources/OAuth.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+var utils = require('../utils');
+
+var oAuthHost = 'connect.stripe.com';
+
+module.exports = StripeResource.extend({
+  basePath: '/',
+
+  authorizeUrl: function(params, options) {
+    // As long as we support Node v4, we have to set defaults like this
```

Comment:
Should probably remove comments like this from public repos 😄

Reply:
Ha, fair enough. I see how that could be read as a knock. :)